Makefile

```diff
@@ -1,2 +1,2 @@
 test:
```

Comment:
Is there any test file left? 
If not, we can replace this with pytest -v without specifying any file